### PR TITLE
Theme development

### PR DESCRIPTION
**Pull request by sevein**
Work in progress! Feedback is more than welcome.

Reference theme: https://github.com/artefactual-labs/atom-theme-corcovado

### CODE REVIEW

File: admin-manual/customization/theming.rst [hunk 1]
```diff
@@ -35,21 +36,58 @@ Customize how an AtoM theme looks
...
 
 
-Theme your own version of AtoM
-------------------------------
+Create a custom theme
+---------------------
 
-1. Make changes to the global layout of the website in
-   apps/qubit/templates/layout.php.
+In most cases, it's probably enough to built a custom theme upon an existing
```
Comment (MikeFE) [inline admin-manual/customization/theming.rst]: s/built/build

File: admin-manual/customization/theming.rst [hunk 2]
```diff
@@ -35,21 +36,136 @@ Customize how an AtoM theme looks
...
+   $ mkdir arCorcovadoPlugin
+   $ git init
+   $ echo "# Corcovado theme" > README.md
+   $ git add README.md
+   $ git commit -m "Initial commit"
+   $ git remote add origin git@github.com:artefactual-labs/atom-theme-corcovado.git
+   $ git push
+
+We've created an empty directory where our plugin is going to be contained,
+made a git repository of it, track our first file and publish our work to
```
Comment (fiver-watson) [inline admin-manual/customization/theming.rst]: keep tense consistent. created... made... switches to track and publish. I would suggest: "...tracked our first file, and published our work to..."

File: admin-manual/customization/theming.rst [hunk 3]
```diff
@@ -35,21 +36,136 @@ Customize how an AtoM theme looks
...
+.. code-block:: bash
+
+   $ cd ~/atom/plugins/arCorcovadoPlugin
+   $ mkdir config
+   $ cd config
+
+Create a new file :file:`arCorcovadoPluginConfiguration.class.php` with the
+following contents:
+
+.. code-block:: php
```
Comment (fiver-watson) [inline admin-manual/customization/theming.rst:94]: might be nice to have a comment or 2 in the code block here explaining what this does?

File: admin-manual/customization/theming.rst [hunk 4]
```diff
@@ -35,21 +36,136 @@ Customize how an AtoM theme looks
...
+       $decoratorDirs[] = $this->rootDir.'/templates';
+       sfConfig::set('sf_decorator_dirs', $decoratorDirs);
+
+       $moduleDirs = sfConfig::get('sf_module_dirs');
+       $moduleDirs[$this->rootDir.'/modules'] = false;
+       sfConfig::set('sf_module_dirs', $moduleDirs);
+     }
+   }
+
+You may also have to clear the Symfony cache, depending on the configuration of
```
Comment (fiver-watson) [inline admin-manual/customization/theming.rst]: link "clear the Symfony cache" to https://www.accesstomemory.org/docs/2.3/admin-manual/maintenance/clear-cache/ ?

File: admin-manual/customization/theming.rst [hunk 5]
```diff
@@ -35,21 +36,136 @@ Customize how an AtoM theme looks
...
+refreshing our browser or doing live reload each time we make a change, but
+that's something that we haven't tried yet.
+
+Symfony plugins allow you to do much more, e.g. you could include our own images,
+override the original templates provided by AtoM or add your owns, inject new
+controllers, filters, signal callbacks and much more. Take a look at our
+`arArchivesCanadaPlugin <https://github.com/artefactual/atom/tree/qa/2.3.x/plugins/arArchivesCanadaPlugin>`_.
+This theme overrides the original homepage template as well as the search box
+and the main header template. Other plugins in the same repository may give you
+more ideas on what are the possibilities - we've built much more than simple
```
Comment (fiver-watson) [inline admin-manual/customization/theming.rst]: ... more ideas of what's possible

File: admin-manual/customization/theming.rst (hunk 3 above)
Comment (sevein) [inline admin-manual/customization/theming.rst:94]: Good idea, done!

**Pull request closed without merging**